### PR TITLE
*: disable batch DMLs by default

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Batching commit a DML statement is useless when TiDB supports larger transaction. 

### What is changed and how it works?
Disable it by default.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: statistics/handle/update.go [hunk 1]
```diff
@@ -564,12 +565,24 @@ func (h *Handle) handleSingleHistogramUpdate(is infoschema.InfoSchema, rows []ch
...
-	sql := fmt.Sprintf("delete from mysql.stats_feedback where table_id = %d and hist_id = %d and is_index = %d", tableID, histID, isIndex)
-	_, err := h.mu.ctx.(sqlexec.SQLExecutor).Execute(context.TODO(), sql)
-	h.mu.ctx.GetSessionVars().BatchDelete = false
-	h.mu.Unlock()
-	return errors.Trace(err)
+	defer func() {
+		h.mu.Unlock()
+	}()
+	for {
+		sql := fmt.Sprintf("delete from mysql.stats_feedback where table_id = %d and hist_id = %d and is_index = %d", tableID, histID, isIndex)
```
Comment (lamxTyler) [inline statistics/handle/update.go]: Why not directly add limit here? The error handling would be simpler.
And maybe 100000 is too large, what about 10000?

**Comment by coocood** [comment]
LGTM

File: statistics/handle/update.go [hunk 2]
```diff
@@ -564,12 +564,19 @@ func (h *Handle) handleSingleHistogramUpdate(is infoschema.InfoSchema, rows []ch
 
 func (h *Handle) deleteOutdatedFeedback(tableID, histID, isIndex int64) error {
 	h.mu.Lock()
-	h.mu.ctx.GetSessionVars().BatchDelete = true
-	sql := fmt.Sprintf("delete from mysql.stats_feedback where table_id = %d and hist_id = %d and is_index = %d", tableID, histID, isIndex)
-	_, err := h.mu.ctx.(sqlexec.SQLExecutor).Execute(context.TODO(), sql)
-	h.mu.ctx.GetSessionVars().BatchDelete = false
-	h.mu.Unlock()
-	return errors.Trace(err)
+	defer func() {
```
Comment (tiancaiamao) [inline statistics/handle/update.go]: defer h.mu.Unlock()

Comment (jackysp) [reply statistics/handle/update.go]: done, PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**